tests/test_jobqueue.py

```diff
@@ -138,7 +173,7 @@ def test_application_weakref(self, bot):
 
     async def test_run_once(self, job_queue):
         job_queue.run_once(self.job_run_once, 0.1)
-        await asyncio.sleep(0.2)
+        await self.wait_for_cbs()
```

Comment:
I don't see the sense in these changes. The tests are no longer testing that the jobs run within the specified time schedule. Now the jobs could run whenever they like.
Not testing on Windows & Mac due to instabilities is more derisable to me than not actually testing the functionality

Reply:
Yes, I could add a `timeout` parameter to `wait_for_cbs()` which ensures the test is still run within the time limit +/- 0.x s. The reason I didn't add this by default is because some tests still use `sleep(x)` which does ensure that

Reply:
Adding a timeout parameter would basically do what we were doing up to now - just more complicated ;)

Reply:
humm, okay I'll revert Experiment 1